core/src/layers/logging.rs

```diff
@@ -105,8 +106,13 @@ impl LoggingLayer {
     /// For example: accessor returns NotFound.
     ///
     /// `None` means disable the log for error.
-    pub fn with_error_level(mut self, level: Option<Level>) -> Self {
-        self.error_level = level;
+    pub fn with_error_level(mut self, level: Option<&str>) -> Self {
+        if let Some(level) = level {
+            let level = Level::from_str(level).expect("parse level must succeed");
```

Comment:
How about returning `Result<Self>` here? It's possible the users building an operator during runtime.